README.md

```diff
@@ -79,7 +79,7 @@ Have a look at [PREINSTALL-README](PREINSTALL-README.md) before proceeding.
 - wayblue-wayfire-main-hardened
 - wayblue-wayfire-nvidia-hardened
 
-##### laptop
+##### laptop<sup>[installer](https://github.com/blue-build/modules/blob/main/modules/bling/installers/laptop.sh), [configs](https://github.com/blue-build/modules/blob/main/modules/bling/50-laptop.conf)</sup>
```

Comment:
This should just link to laptop-bling.yml since this might not be the sole change for laptops in the future.

Also, you only added the link in one of the two places where there's a laptop header.

Reply:
Thanks for your review. Given the size of this PR, I think it is probably more clean to just close this and open a new PR.

Reply:
@czhang03 you can just modify it and then squash into one commit.